warmup.py

```diff
@@ -225,7 +227,12 @@ class RadialDistribution:
 
     """
     def __init__(self, rmax, dr, rmin=0.0):
-        pass
+        self.rmax = rmax
+        self.dr = (rmax - rmin) / np.round((rmax - rmin) / dr)
+        self.rmin = rmin
+        self.edges = np.arange(rmin, rmax + (dr / 2), dr)
+        self.centers = np.arange(rmin + (dr / 2), rmax + (dr / 2), dr)
+        self.rdf_var = np.zeros(len(self.centers))
```

Comment:
For variables that are "private" version of properties (like `self.rdf`), you can use a leading underscore like `self._rdf`.